core/regexp/options_spec.rb

```diff
@@ -22,6 +22,10 @@
       (/cat/xi.options & Regexp::EXTENDED).should_not == 0
     end
   end
+  
+  it "retrieves options on an empty expression" do
```

Comment:
maybe `returns 0 for a Regexp literal without options` (and then just add another example)